src/Editor/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+set(LIQUIUM_EDITOR "Editor")
```

Comment:
could add file glob for automatic source file discovery but I guess it's not need for now.